curl/inc/leatherman/curl/client.hpp

```diff
@@ -164,6 +164,13 @@ namespace leatherman { namespace curl {
          */
         void set_client_cert(std::string const& client_cert, std::string const& client_key);
 
+        /**
+         * Set and limit what protocols curl will support
+         * @param client_protocols bitmask of CURLPROTO_*
+         *        (see more: http://curl.haxx.se/libcurl/c/CURLOPT_PROTOCOLS.html)
+         */
+        void set_supported_protocols(long client_protocols);
```

Comment:
Is this expected to be a 32-bit or 64-bit type? We should pretty much never use long, as it has different size on Windows vs other OS.